pycaret/internal/preprocess/time_series/forecasting/preprocessor.py

```diff
@@ -152,11 +157,13 @@ def _add_transformation_steps(self, transform: str, target: bool = True):
 
         if isinstance(transform, str):
             transform_dict = {
-                "box-cox": BoxCoxTransformer(),
-                "log": LogTransformer(),
-                "sqrt": SqrtTransformer(),
-                "exp": ExponentTransformer(),
-                "cos": CosineTransformer(),
+                "box-cox": BoxCoxTransformer,
+                "log": LogTransformer,
+                "sqrt": SqrtTransformer,
+                "exp": ExponentTransformer,
+                "cos": CosineTransformer,
+                "scaledlogit" : ScaledLogitTransformer
```

Comment:
I think the `ScaledLogitTransformer` should be done after imputation and separate from other transformations

So someone might want to limit the data but also apply log/box-cox transformations if residual variance needs to be satisfied.

What do you think?